### ISSUE TITLE
GraphQL returns incorrect block height for block queries when there are over 100 blocks

### ISSUE THREAD

**Issue by DanMacDonald** [textury/arlocal#128]
```js
import Arweave from "arweave";

const arweave = Arweave.init({
	host: "127.0.0.1",
	port: 1984,
	protocol: "http",
});

(async () => {
	// Query the blockheight via GQL
	let currentBlockHeight = await getArweaveBlockheight();
	console.log(`initial blockHeight: ${currentBlockHeight}`);

	// Mint 110 blocks
	for (let i = 0; i < 110; i++) {
		await arweave.api.get(`mine`);
	}

	// Query the blockheight via GQL again
	currentBlockHeight = await getArweaveBlockheight();
	console.error(`queried blockHeight: ${currentBlockHeight}`);
	
	// Now check the blockheight via the info endpoint
	const response = await arweave.api.get("");
	console.log(`actual blockHeight: ${response.data.height}`);
	console.log("done");
})();

async function getArweaveBlockheight(): Promise<number> {
	const queryObject = { query: `query{ blocks(sort: HEIGHT_DESC, first:100 ){ edges{ node{ height } } } }` }
	const result = await arweave.api.post(`/graphql`, queryObject)
	const edge = result.data.data.blocks.edges[0];
	if (edge) {
		const arweaveBlockHeight = edge.node.height;
		return arweaveBlockHeight;
	}
	return -1;
}
```
running this test with a fresh instance of `arlocal` produces the following output...
![image](https://user-images.githubusercontent.com/3269261/180117051-b94bf8ae-5b46-47c2-b341-09f7f12548dc.png)

## Expected result
A block height GQL query returns the same height as the info endpoint when there are more than 100 blocks.

**Comment by fuadop** [comment]
Thanks for the bug report @DanMacDonald. It has been fixed on the latest version (1.1.52) 🎉 . Please update your arlocal and verify this fix works.

**Comment by DanMacDonald** [comment]
Love it @fuadop 🥳 Keep kicking 🍑!

Tested locally and now I'm seeing the expected blockHeight 🙌
